CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ Which is based on [Keep A Changelog](http://keepachangelog.com/)
 
 
 ## [Unreleased]
+### Fixed
```

Comment:
```suggestion
### Added
```
I know this might sound a bit odd but I consider adding support for new versions of software to be enhancements as opposed to a fix. Really wish more projects would follow semver and bump major versions when they choose to change the behavior of the software in a way that breaks compatibility with previous expectations.